src/ImageTagEditor.js

```diff
@@ -49,7 +49,12 @@ export default class RedaxtorImageTag extends Component {
             url: this.targetImg.src,
             alt: this.targetImg.alt || "",
             width: this.targetImg.width,
-            height: this.targetImg.height
+            height: this.targetImg.height,
+            pieceData: {
```

Comment:
please rename it to pieceRef and add "dataset" to data passed
dataset is user-specified data and may be usefull in abstract scenario